juspay/hyperswitch
# fix: merchant_connector_id null in KV flow

## Type of Change


## Description

Added or function to use the source merhchant_connector_id if the updatable one is null

### Additional Changes





## Motivation and Context

This fixes merchant_connector_id being null PaymentsResponse

## How did you test it?

- Create a Payment with KV merchant.
- Confirm the Payment.
- Check if `merchant_connector_id` is present in response or not

## Checklist
